tests/integration/a2a/test_push_notification.py

```diff
@@ -196,7 +201,10 @@ async def webhook_handler(request: Request) -> JSONResponse:
             if hasattr(response_1.root, "error"):
                 msg = f"Error: {response_1.root.error.message}, Code: {response_1.root.error.code}, Data: {response_1.root.error.data}"
                 raise RuntimeError(msg)
-            task_id = response_1.root.result.id
+            if isinstance(response_1.root.result, Task):
```

Comment:
Alternatives are considered here, since messages could also be sent. But a sanity check would be ok.